tests/TaintTest.php

```diff
@@ -2285,6 +2285,25 @@ function sinkNotWorking($sink) : string {}
                     echo sinkNotWorking($_GET["taint"]);',
                 'error_message' => 'TaintedHtml',
             ],
+            'taintEscapedInTryMightNotWork' => [
+                '<?php
+                    /** @psalm-taint-escape html */
+                    function escapeHtml(string $arg): string
+                    {
+                        return htmlspecialchars($arg);
+                    }
+
+                    $tainted = $_GET["foo"];
+
+                    try {
+                        $tainted = escapeHtml($tainted);
+                    } catch (Throwable $_) {
+                    }
+
+                    echo $tainted;
+                ',
+                'error_message' => 'TaintedHtml',
+            ],
```

Comment:
This test actually passes already, but I left it here anyway in case `try/catch/finally` analysis improves in the future and accidentally breaks it.